pkgs/by-name/un/universal-android-debloater/package.nix

```diff
@@ -0,0 +1,63 @@
+{ android-tools
+, clang
+, expat
+, fetchFromGitHub
+, fontconfig
+, freetype
+, lib
+, libglvnd
+, makeWrapper
+, mold
+, pkg-config
+, rustPlatform
+, xorg
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "universal-android-debloater";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "Universal-Debloater-Alliance";
+    repo = pname;
+    rev = version;
+    hash = "sha256-8s4/lAekW2glz4lH79UtbziToytT53m5wQGTVMBAqMU=";
+  };
+
+  cargoHash = "sha256-fMW9CmDyJ77PIuJ6QGI8nNZsuAZwkL9xf3xbbX13HKw=";
+
+  buildInputs = [
+    expat
+    fontconfig
+    freetype
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+    mold
+    pkg-config
+  ];
+
+  nativeCheckInputs = [
+    clang
+  ];
+
+  preCheck = ''
+    export HOME="$(mktemp -d)"
+  '';
+
+  postInstall = ''
+    wrapProgram $out/bin/uad_gui \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ fontconfig freetype libglvnd xorg.libX11 xorg.libXcursor xorg.libXi xorg.libXrandr ]} \
```

Comment:
Can we avoid LD_LIBRARY_PATH somehow?

Reply:
This program depends on `iced_winit`, which depends on `winit`, which depends on `x11-dl`, which uses dlopen in https://github.com/AltF02/x11-rs/blob/ff57a942e5a4d35d8660362a8ca2ac0c02fa4852/x11-dl/src/link.rs. I have no idea how to patch Rust dependencies in Nixpkgs, so `LD_LIBRARY_PATH` is the simplest way around that.

Reply:
Yeah, looked into patching `winit` dependency, I don't think it's practical to change what https://github.com/rust-windowing/winit/blob/ca1674519ab3d8df4ce231fe018196a3981c7dea/src/platform_impl/linux/x11/xdisplay.rs does.

Reply:
> Can we avoid LD_LIBRARY_PATH somehow?

This suggestion is crazy and will probably blow up lots of things started from that shell.

I think we can keep it for the wrapper if there is no good alternative.